contracts/proxies/ProxyFactory.sol

```diff
@@ -56,4 +56,26 @@ contract ProxyFactory {
             }
         emit ProxyCreation(proxy);
     }
+
+    /// @dev Allows to get the address for a new proxy contact created via `createProxyWithNonce`
```

Comment:
Should we mention that this is only required if you use an initializer that would revert.